### ISSUE TITLE
rh_ip templates cannot set DELAY=0

### ISSUE THREAD

**Issue by tjstansell** [saltstack/salt#21620]
There are plenty of examples of using network.managed and setting `delay: 0`, but this appears to be broken on redhat because of the template logic.  The code in question does this:

```
{% if delay %}DELAY="{{delay}}"{% endif %}
```

Unfortunately, when delay is set to 0, this fails the test and the DELAY line is never added.

If this is changed to:

```
{% if delay or delay == 0 %}DELAY="{{delay}}"{% endif %}
```

then it seems to do what we want.

**Comment by rallytime** [comment]
Thanks for filing this bug and for fixing it, @tjstansell! :) Should this issue be closed now?

**Comment by tjstansell** [comment]
Yeah, we can close this one.  Does this specifically need to get requested to get merged into 2015.2 and develop too? or does that happen "automatically" when a PR is for an older branch like this?

**Comment by rallytime** [comment]
@tjstansell It does not need to be requested. We merge 2014.7 --> 2015.2 and then 2015.2 --> develop on an almost daily basis. So it happens "automatically". :)
